aspen/http/mapping.py

```diff
@@ -86,39 +91,3 @@ def ones(self, *names):
             if n not in lowered:
                 lowered.append(n)
         return [self[name] for name in lowered]
-
-
-class CaseInsensitiveMapping(Mapping):
```

Comment:
We're fine removing this (and somehow making `Mapping` case insensitive) because *all* of our HTTP use cases are case insensitive, yes?